app/static/css/style.css

```diff
@@ -8,6 +8,10 @@ body {
   background: rgb(114, 114, 114);
 }
 
+#app {
+  display: grid;
```

Comment:
on my phone the nav bar was not going all the way across. This had fixed it, but now I cant duplicate the problem lol.

We can take it out for now, if I run into the issue again I will do a screen capture and share with you.

Reply:
Here you go, with and without.


![image](https://user-images.githubusercontent.com/22175951/93928954-1c43b400-fce9-11ea-864e-8947851b014d.png)

![image](https://user-images.githubusercontent.com/22175951/93929006-2d8cc080-fce9-11ea-8b17-2598971598e1.png)

Reply:
Ah, that's an unrelated issue then. I've filed #263 to track it. At first glance, it seems like a problem with the keyboard history panel rather than the `#app` CSS rules. I want to be conservative about setting CSS rules in the root element for the page.